UPDATE 3: Add ratings to Info Wall, new plan

mechevarria opened the pull request:
Added the imdb, metacritic, rotten tomatoes, etc. ratings to the `Info Wall, Tri-Panel` and `Poster` Views. Added a param to keep the ratings **off** `Media Info` since `Media Info 2` has that information.

Before:
![before](https://user-images.githubusercontent.com/2657771/74117854-05e84e00-4b87-11ea-999a-d20b7515b649.png)

After:
![after](https://user-images.githubusercontent.com/2657771/74117860-0bde2f00-4b87-11ea-87b1-0eeb4ea08f36.png)
![after2](https://user-images.githubusercontent.com/2657771/74117861-0e408900-4b87-11ea-86e3-471f0151cd8c.png)
![after3](https://user-images.githubusercontent.com/2657771/74117870-14366a00-4b87-11ea-915d-d4ca30a897ac.png)

mechevarria committed:
Bug Fix - Remove extra >

mechevarria committed:
Merge remote-tracking branch 'upstream/master'

mechevarria committed:
Add ratings to Info Wall, Tri-Panel and Poster

jurialmunkey commented:
**Text goes beyond view bounds**

![Honeyview_screenshot474](https://user-images.githubusercontent.com/4618719/74322658-5d312d00-4dd8-11ea-946c-ef0366fc0d2b.jpg)


**Ratings cut-off when tmdb enabled, text cut-off, text row height doesn't match with view bounds**

![Honeyview_screenshot476](https://user-images.githubusercontent.com/4618719/74322660-5dc9c380-4dd8-11ea-9b1a-e22909991b2f.jpg)

**Text cut-off and text row height doesn't match with view bounds**
![Honeyview_screenshot473](https://user-images.githubusercontent.com/4618719/74322655-5c000000-4dd8-11ea-907c-bdaf4ecf605f.jpg)

jurialmunkey commented:
The text spacing from view margins is consistent across the skin. This is how the text spacing from the view margin should appear:

![Honeyview_screenshot477](https://user-images.githubusercontent.com/4618719/74323750-1a705480-4dda-11ea-85f4-e8bf3fee7c84.jpg)

Notice how the spacing from the baseline of the last row of text to the red line of the view margin is the same spacing (or very close to) the spacing between the ascent of last row of text to the baseline of the second last row of text. This spacing should be consistent throughout the skin.

mechevarria committed:
Fix text overflow and make spacing consistent

mechevarria committed:
Remove ratings from tri-panel for cutoff

mechevarria commented:
I updated the spacing so the text isn't cutoff and spacing is consistent. The strategy I used was to group the ratings as visually part of the title and subtitle and then have a consistent space between the fanart/title and plot. Removed ratings from tri-panel completely because of the cutoff

![fixed1](https://user-images.githubusercontent.com/2657771/74361359-c42cf280-4d94-11ea-8cce-4ca2be3ea0f4.png)
![fixed2](https://user-images.githubusercontent.com/2657771/74361361-c42cf280-4d94-11ea-8a24-5dbd735df56a.png)

jurialmunkey commented:
I think the spacing from info line to ratings should be consistent with info dialog.

These are the reasons why I haven't implemented ratings in these views previously - because it just introduces stylistic inconsistencies or makes things too cramped.

Plot and info line are the more important details imo. That's why I've preferenced them.

mechevarria committed:
backout ratings change on poster view

mechevarria committed:
only changing info wall for now

mechevarria committed:
Play with spacing of ratings to match other views

mechevarria commented:
Alright, backed out all the changes to the poster view and just focusing on getting this working for `Info Wall`. Adjusted the spacing so that it is consistent from the fanart to title/info to ratings to plot. That way things look the same as in `Media Info 2`

![after1](https://user-images.githubusercontent.com/2657771/74392949-a97b6d80-4dd6-11ea-931f-fd065b3cb9a4.png)
![after2](https://user-images.githubusercontent.com/2657771/74392950-aa140400-4dd6-11ea-92bc-633894926238.png)

mechevarria committed:
put a little more space from the fanart to title so things are evenly spaced. Was noticeable in Advanced Emulator Launcher, a view that does not have ratings

brentonv commented:
Damn I was hoping you would get poster view approved, it looks great!

jurialmunkey commented:
Okay looking better. I'll give it a test when I'm back home from work.

mechevarria commented:
Thumbs up or thumbs down on the new changes? I wanted to go back to some AEL additions but didn't want to mix that up with these changes or get into branch gymnastics

brentonv commented:
Please don't give up on poster wall @mechevarria. I've been so excited to see this change approved and you are so close.

jurialmunkey commented on 1080i/Includes_View_52_Showcase.xml:
```diff
@@ -139,13 +141,19 @@
                             <align>left</align>
                             <aligny>top</aligny>
-                            <height>200</height>
+                            <height>163</height>
```
This value should be 160. Each line of the plot box is 40 px.

jurialmunkey commented on 1080i/Includes_View_52_Showcase.xml:
```diff
@@ -114,14 +115,15 @@
                     </include>
                     <control type="grouplist">
-                        <top>451.435</top>
+                        <top>440.435</top>
```
I'm not too keen on this change to the spacing as it will now be inconsistent with other views. I'm not sure where else you would get the extra pixels from though...

mechevarria commented:
Now that I have noticed the rating in other places in the skin, keeping the spacing consistent is the right way to go.  Here is another idea if you are open to it. Keep the existing `Info Wall` and `Poster` as is. I can make `Info Wall 2` and `Poster 2`. The difference being that the plot details are replaced with the ratings and nothing else. 
The idea being that it won't really work trying to read the movie plot in a squished space, so don't bother. Some people pick movies based on ratings and others by plot. If the ratings look good, you can also select the movie and then read the entire plot on the movie details page.

Should I try it out?

jurialmunkey commented:
Yeah I think a secondary view would be ideal anyway as then users have a choice (other option is some sort of toggled setting) 

I like the idea of ratings only. My only concern is I think you might end up with too much blank space at the bottom (but not enough for plot). Maybe tag line or genre or something like that might be useful there as a one liner. 

Sorry to be such a pain about all of this! These are all the reasons why I haven't implemented something similar myself. If you can find a way that fits the design specs of the skin, then I'm more than happy to merge!

mechevarria commented:
Going to close this one out and try in a new fork
